Moli/Core/Screens/Home/Live/HostItemView.swift

```diff
@@ -28,7 +46,8 @@ struct HostItemView: View {
                             .font(.title)
                             .textCase(.uppercase)
                             .foregroundColor(.primaryDark)
-                    )
+                    ).frame(minWidth: 0, maxWidth: .infinity)
+                    .frame(height: getFrameHeight())
```

Comment:
:art: Du ska nog få samma beteende från att bara ha maxWidth
```suggestion
                    )
                    .frame(maxWidth: .infinity)
                    .frame(height: getFrameHeight())
```